### ISSUE TITLE
No Result from get_post_meta( ) in post_user_likes() function

### ISSUE THREAD

**Issue by fro35i** [JonMasterson/WordPress-Post-Like-System#18]
Hi Jon,

WP function update_post_meta( $post_id, "_user_liked", $post_users ) will never be called, because $post_users as result of the function post_user_likes( $user_id, $post_id, $is_comment ) is always empty. So the meta key "_user_liked" will never be added in WP.
That's why you won't be able to check, if the user has already liked or not.

Am I right or do I miss something?
Thanks

**Comment by JonMasterson** [comment]
Hi fro35i, I removed the failsafe array check from post_user_likes() and post_ip_likes(). The failsafe checked if $post_users was an array, and if not, made it an array. This failsafe is important when a post/comment has no previous likes. Without any likes, the meta is empty (is not an array), and the user's IP/ID is not added to the $post_users array. This is why it returned an empty value. I hadn't noticed it in testing, because my testing environment only contained posts that had previously been liked. My apologies, I just fixed it. Thanks for the heads up.
